### PR TITLE
Archive: updates to reflect archival status.

### PR DESCRIPTION
**Pull request by zsistla**

### CODE REVIEW

**Review by Ak-x — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # New Relic C Agent Release Notes #
 
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```
Comment (lavarou) [inline CHANGELOG.md]: `option to use OpenTelemetry` link (https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) gives me 404 - Not Found. Shouldn't this link point to https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,45 +1,18 @@
...
 
-## How to Contribute
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
+
+## Best Practices for Modifying Project Forks
 
 1. Read [this contributing file](CONTRIBUTING.md). (Good job!)
 1. Read our [code of conduct](CODE_OF_CONDUCT.md).
 1. Submit a [pull request](#pull-request-guidelines) or [issue](#filing-issues--bug-reports). For pull requests, please also:
```
Comment (lavarou) [inline CONTRIBUTING.md]: Is this section applicable for archived projects? Is New Relic going to review and merge any incoming PRs?

File: GUIDE.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 This is the New Relic C SDK! If your application does not use other New Relic APM agent languages, you can use the C SDK to take advantage of New Relic's monitoring capabilities and features to instrument a wide range of applications.
 
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```
Comment (lavarou) [inline GUIDE.md]:
```suggestion
As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,45 +1,18 @@
 # Contributing to the C SDK
 
-Thanks for your interest in contributing to the New Relic C SDK! We look forward to engaging with you.
+Thanks for your interest in contributing to the New Relic C SDK! 
 
-## How to Contribute
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```
Comment (lavarou) [inline CONTRIBUTING.md:6]:
```suggestion
As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```


```

File: CHANGELOG.md (hunk 1 above)
Comment (lavarou) [reply CHANGELOG.md]:
```suggestion
As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```

File: README.md [hunk 5]
```diff
@@ -8,6 +8,9 @@ monitoring capabilities and features to instrument a wide range of applications.
 
 [![Build Status](https://travis-ci.org/newrelic/c-sdk.svg?branch=master)](https://travis-ci.org/newrelic/c-sdk)
 
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```
Comment (lavarou) [inline README.md:12]:
```suggestion
As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```

File: README.md [hunk 6]
```diff
@@ -68,23 +71,6 @@ Or, just to compile them:
...
-
-* [New Relic Documentation](https://docs.newrelic.com/docs/c-sdk-table-contents): Comprehensive guidance for using our platform
-* [New Relic C SDK documentation on GitHub](https://newrelic.github.io/c-sdk/index.html)
-* [New Relic C SDK Guide](https://github.com/newrelic/c-sdk/blob/master/GUIDE.md)
-* [New Relic C SDK distributed tracing example](https://github.com/newrelic/c-dt-example)
-* [New Relic Community](https://discuss.newrelic.com/c/build-on-new-relic/Open-Source-Agents-SDKs): The best place to engage in troubleshooting questions
-* [New Relic Developer](https://developer.newrelic.com/): Resources for building a custom observability applications
-* [New Relic University](https://learn.newrelic.com/): A range of online training for New Relic users of every level
-* Use your preferred search engine to find other New Relic resources.
-
```
Comment (lavarou) [inline README.md:87]: Is New Relic going to remove all C SDK documentation from their website? If no, then I'd suggest to leave these links in and only remove the link to `New Relic Community`.

File: README.md [hunk 7]
```diff
@@ -94,9 +80,7 @@ For more information, review [New Relic’s General Data Privacy Notice](https:/
 
 ## Contribute
 
-We encourage your contributions to improve the C SDK! Keep in mind when you submit your pull request, you'll need to sign the CLA via the click-through using CLA-Assistant. You only have to sign the CLA one time per project.
-
-If you have any questions, or to execute our corporate CLA, required if your contribution is on behalf of a company,  please drop us an email at opensource@newrelic.com.
+We encourage your experiments to improve the C SDK in your personal forks.
```
Comment (lavarou) [inline README.md]: I'd suggest reiterating what the `Archived` banner says:
> This project is read-only and available for cloning only. We encourage you to experiment with it in your fork however please keep in mind that there is neither a maintainer team nor support.

File: ROADMAP.md [hunk 8]
```diff
@@ -1,24 +1,8 @@
 # C SDK Roadmap
 
+## Archival Notice ##
+As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://github.com/newrelic/docs-website/blob/develop/docs/apm/agents/c-sdk/get-started/otel_cpp_example) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```
Comment (lavarou) [inline ROADMAP.md]:
```suggestion
As of June 30 2022 we are archiving the C-SDK project. Please explore the [option to use OpenTelemetry](https://docs.newrelic.com/docs/apm/agents/c-sdk/get-started/otel_cpp_example/) as an alternative to send C++ telemetry data to New Relic. For more details, see our [Explorers Hub announcement](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```

**Review by lavarou — CHANGES_REQUESTED** [review]
This looks good. There are a few typos in the URLs that show `404 - Page not found` page instead of intended resource.

**Review by lavarou — APPROVED** [review]

**Pull request merged**